frameworks/hdfs/universe/config.json

```diff
@@ -13,13 +13,18 @@
         "user": {
           "type": "string",
           "description": "The linux user used to run the scheduler and all executors.",
-          "default": "root"
+          "default": "nobody"
         },
         "principal" : {
           "description":"The principal for the HDFS service instance.",
           "type":"string",
           "default":""
         },
+        "mesos_api_version": {
```

Comment:
Looks like there's supposed to be a `"MESOS_API_VERSION": "{{service.mesos_api_version}}"` in marathon.json. See examples in kafka, helloworld, cassandra.